modules/utxo-lib/src/bitgo/transaction.ts

```diff
@@ -9,32 +9,44 @@ import { DashTransactionBuilder } from './dash/DashTransactionBuilder';
 import { ZcashTransactionBuilder } from './zcash/ZcashTransactionBuilder';
 import { ZcashNetwork, ZcashTransaction } from './zcash/ZcashTransaction';
 
-export function createTransactionFromBuffer(
+export function createTransactionFromBuffer<TNumber extends number | bigint = number>(
   buf: Buffer,
   network: Network,
+  amountType: 'number' | 'bigint' = 'number',
```

Comment:
Change in required parameters. Please record as BREAKING CHANGE in commits.

Reply:
See first conversation